Specification.md

```diff
@@ -40,40 +40,78 @@ seq=11                                  \
 
 ## 2. Format and Naming
 
-**Sync Interest Name**: `/<group-prefix>/v=2/<parameters-digest>`
+**State Vector Data Name**: `/<group-prefix>/v=3`
 
-The State Vector is encoded in TLV format and included in the `ApplicationParameters` of the Sync Interest.
-The State Vector SHOULD be the first TLV block in the `ApplicationParameters`.
-The Interest Lifetime for Sync Interests is 1 second.
+The State Vector is encoded in the content of a signed NDN Data packet.
 
-**Data Interest Name**: `/<node-prefix>/<group-prefix>/<seq-num>`
+**Sync Interest Name**: `/<group-prefix>/v=3/<parameters-digest>`
+
+The State Vector Data is included in the `ApplicationParameters` of the Sync Interest.
+The Interest Lifetime for the Sync Interest is 1 second.
```

Comment:
is this a "should" or a "must"?

Reply:
that's what I'm asking... how strict is the requirement of 1s lifetime?

Reply:
Ah okay. Yeah I'll change that to should, don't see a hard requirement 👍🏻